charts/pvc-autoresizer/README.md

```diff
@@ -61,6 +61,8 @@ helm upgrade --create-namespace --namespace pvc-autoresizer -i pvc-autoresizer -
 | podMonitor.relabelings | list | `[]` | RelabelConfigs to apply to samples before scraping. |
 | podMonitor.scheme | string | `"http"` | Scheme to use for scraping. |
 | podMonitor.scrapeTimeout | string | `""` | The timeout after which the scrape is ended |
+| serviceAccount.automountServiceAccountToken | bool | `true` | Controls the automatic mounting of ServiceAccount API credentials. |
+| serviceAccount.create | bool | `true` | Creates a ServiceAccount for the controller deployment. |
```

Comment:
Could you rename this field for consistency with `podMonitor.enabled`?
```suggestion
| serviceAccount.enabled | bool | `true` | Creates a ServiceAccount for the controller deployment. |
```

Reply:
Updated!